sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsRestClient.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.AI.TextAnalytics.Models;
+using Azure.Core;
+
+namespace Azure.AI.TextAnalytics
+{
+    internal partial class TextAnalyticsRestClient
+    {
+        internal HttpMessage CreateHealthStatusNextPageRequest(string nextLink, bool? showStats)
+        {
+            var message = _pipeline.CreateMessage();
+            var request = message.Request;
+            request.Method = RequestMethod.Get;
+            var uri = new RawRequestUriBuilder();
+            uri.AppendRaw(endpoint, false);
+            uri.AppendRaw("/text/analytics/v3.1-preview.3", false);
+            uri.AppendRawNextLink(nextLink, false);
+            if (showStats != null)
```

Comment:
What about stringIndexType ?

Reply:
Index type is only for the POST operation. This one is GET and doesn't require it